setup.py

```diff
@@ -78,5 +77,4 @@ def get_test_requires():
           'zope.testbrowser >= 4.0, < 5.0',
       ],
       include_package_data=True,
-      zip_safe=False,
-     )
+      zip_safe=False,)
```

Comment:
I would rather see the bracket on the beginning of th new line in case something has to be added afterwards.